src/java_bytecode/mz_zip_archive.cpp

```diff
@@ -25,6 +25,14 @@ class mz_zip_archive_statet final:public mz_zip_archive
     if(MZ_TRUE!=mz_zip_reader_init_file(this, filename.data(), 0))
       throw std::runtime_error("MZT: Could not load a file: "+filename);
   }
+
+  explicit mz_zip_archive_statet(const void *pMem, size_t size):
+    mz_zip_archive({ })
+  {
+    if(MZ_TRUE!=mz_zip_reader_init_mem(this, pMem, size, 0))
+      throw std::runtime_error("MZT: Could not load data from memory");
```

Comment:
Does the user know what `MZT` is?

Reply:
MZT is as existing messages in this file.
Do you want these changed?